openaq_api/openaq_api/v3/routers/owners.py

```diff
@@ -41,10 +49,20 @@ def where(self) -> str:
         Returns:
             string of WHERE clause
         """
-        return "entities_id = :owners_id"
+        return "(owner->>'id')::int = :owners_id"
 
 
-class OwnersQueries(Paging):
+class OwnersQueries(
+    Paging,
+    RadiusQuery,
+    BboxQuery,
+    ProviderQuery,
+    OwnerQuery,
+    CountryIdQuery,
+    CountryIsoQuery,
+    MobileQuery,
+    MonitorQuery,
```

Comment:
Not sure that this makes sense to me. to me this reads as "show me owners who have a monitor (or air sensors)" that wouldnt really filter things down much. I would vote to remove this and mobilequery

Reply:
They have been removed.